traitcuration/traits/models.py

```diff
@@ -0,0 +1,66 @@
+from django.db import models
+
+
+class Trait(models.Model):
+    name = models.CharField(max_length=200)
```

Comment:
If ClinVar one day introduces a trait name longer than 200 characters, what will happen under the current model? I'm guessing we should introduce a check into the import process (once that is implemented) to avoid a hard crash

Reply:
Yes, I can add that check. Should I increase the maximum length as well?

Reply:
No, I think the current lenght limit is fine. If ClinVar actually adds a trait which name is monstrously large, then I _want_ the import to fail and to notify us in this way, because who knows what they can put in place of a trait name :)

Since this PR only concerns the prototype, I've added a separate checkbox to #14 description to implement this check